### PR TITLE
refactor: all page sprint 1

### PR DESCRIPTION
**Pull request by C1021G1datnvn**

### CODE REVIEW

File: src/components/Campaign/Campaign.tsx [hunk 1]
```diff
@@ -7,7 +7,6 @@ import { CampaignType } from '@types';
 import { useGetListCampaign } from '../../hooks';
 import { SearchBar } from '../SearchBar/SearchBar';
 import './Campaign.css';
-import { UpdateCampaign } from './UpdateCampaign';
 import { useNavigate } from 'react-router-dom';
 import { useDeleteCampaign } from 'hooks/useDeleteCampaign';
 import { ExclamationCircleFilled } from '@ant-design/icons';
```
Comment (stunited-thanhle) [inline src/components/Campaign/Campaign.tsx]: move up

File: src/components/Campaign/Campaign.tsx [hunk 2]
```diff
@@ -116,21 +100,23 @@ export const Campaign = () => {
...
+								{/* <Link to={'/update-campaign/' + record.id}> */}
 								<svg
 									xmlns="http://www.w3.org/2000/svg"
 									viewBox="0 0 24 24"
 									className="icons-check"
 									onClick={(e) => {
 										e.preventDefault();
 										e.stopPropagation();
-										showModal(record);
+										navigate('/update-campaign/' + record.id);
```
Comment (stunited-thanhle) [inline src/components/Campaign/Campaign.tsx:99]: how about error?

File: src/components/WorkSpace/CreateWorkSpace.tsx [hunk 3]
```diff
@@ -1,26 +1,41 @@
...
+	const navigate = useNavigate();
+
+	useEffect(() => {
+		if (registrationSuccess) {
+			setTimeout(() => {
+				navigate('/workspace');
+			}, 1000);
+		}
+	}, [navigate, registrationSuccess]);
+
```
Comment (stunited-thanhle) [inline src/components/WorkSpace/CreateWorkSpace.tsx]: why use useEffect and setTimeout to check navigate ??

File: src/components/Campaign/UpdateCampaign.tsx [hunk 4]
```diff
@@ -1,66 +1,78 @@
...
-	};
+	const navigate = useNavigate();
+
+	useEffect(() => {
+		if (registrationSuccess) {
+			setTimeout(() => {
+				navigate('/campaign');
+			}, 10);
+		}
+	}, [navigate, registrationSuccess]);
```
Comment (stunited-thanhle) [inline src/components/Campaign/UpdateCampaign.tsx]: why use useEffect and setTimeout to check navigate ??

**Review by demosofa — APPROVED** [review]

**Pull request merged**